conf/application.conf

```diff
@@ -119,7 +119,7 @@ keys {
 
 mongodb {
   uri = "mongodb://localhost:27017/transits-movements-trader-at-departure"
-  timeToLiveInSeconds = 2419200
+  timeToLiveInSeconds = 120
```

Comment:
Should this still be 120 seconds?  Was 120 just for local testing?

Reply:
Oh, yeah this PR is a draft, i didnt realise it would create the notifications etc.